diff --git a/effectinfo.txt b/effectinfo.txt
index b2544fa..b7ae6e1 100644
--- a/effectinfo.txt
+++ b/effectinfo.txt
@@ -8253,3 +8253,94 @@ effect arc_bolt_explode
 	tex 40 40
 	velocityjitter 224 224 224
 	velocityoffset 0 0 80
+
+//Small splash of water
+// main water
+effect water_splash_small
+	type alphastatic
+	tex 76 76
+	count 7
+	size 6 16
+	sizeincrease -10
+	alpha 100 450 656
+	velocityjitter 10 10 250
+	velocitymultiplier 0.6
+	airfriction 4
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -150 150
+	gravity 1.5
+// small water drops
+effect water_splash_small
+	type spark
+	tex 76 76
+	count 4
+	size 2 1
+	sizeincrease -5
+	alpha 50 145 200
+	velocityjitter 210 210 350
+	velocitymultiplier 0.2
+	airfriction 1
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -150 150
+	gravity 0.4
+	stretchfactor 2
+// water mist
+effect water_splash_small
+	type smoke
+	tex 75 75
+	count 4
+	size 6 26
+	sizeincrease 25
+	alpha 100 150 256
+	velocityjitter 150 150 250
+	velocitymultiplier 0.4
+	airfriction 4
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -15 15
+	gravity 0.9
+// Large splash of water
+// main water
+effect water_splash_large
+	type alphastatic
+	tex 76 76
+	count 15
+	size 15 25
+	sizeincrease -20
+	alpha 100 250 600
+	velocityjitter 150 150 150
+	velocitymultiplier 0.2
+	airfriction 1
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -150 150
+	gravity 0.5
+	originjitter 21 21 1
+// small water drops
+effect water_splash_large
+	type spark
+	tex 76 76
+	count 4
+	size 2 1
+	sizeincrease -5
+	alpha 50 145 200
+	velocityjitter 210 210 350
+	velocitymultiplier 0.3
+	airfriction 1
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -150 150
+	gravity 0.4
+	stretchfactor 2
+// water mist
+effect water_splash_large
+	type smoke
+	tex 75 75
+	count 10
+	size 6 26
+	sizeincrease 35
+	alpha 200 250 356
+	velocityjitter 180 180 250
+	velocitymultiplier 0.4
+	airfriction 4
+	color 0xd3e5ea 0xa6eaff
+	rotate 0 360 -15 15
+	gravity 0.9
+	originjitter 21 21 1
diff --git a/particles/particlefont.tga b/particles/particlefont.tga
index a233482..9ad68cd 100644
Binary files a/particles/particlefont.tga and b/particles/particlefont.tga differ
diff --git a/qcsrc/common/effects/effectinfo.inc b/qcsrc/common/effects/effectinfo.inc
index 56303a4..77674e4 100644
--- a/qcsrc/common/effects/effectinfo.inc
+++ b/qcsrc/common/effects/effectinfo.inc
@@ -8873,3 +8873,151 @@ SUB(arc_bolt_explode) {
 	MY(velocityjitter) = '224.0 224.0 224.0';
 	MY(velocityoffset) = '0.0 0.0 80.0';
 }
+
+// small splash of water
+DEF(water_splash_small);
+// main water
+SUB(water_splash_small) {
+	MY(airfriction) = 4;
+	MY(alpha_min) = 100;
+	MY(alpha_max) = 450;
+	MY(alpha_fade) = 656;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 7;
+	MY(gravity) = 1.5;
+	MY(sizeincrease) = -10;
+	MY(size_min) = 6;
+	MY(size_max) = 16;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -150;
+	MY(spin_max) = 150;
+	MY(tex_min) = 76;
+	MY(tex_max) = 76;
+	MY(type) = "alphastatic";
+	MY(velocityjitter) = '0.0 10.0 250.0';
+	MY(velocitymultiplier) = 0.6;
+}
+// small water drops
+SUB(water_splash_small) {
+	MY(airfriction) = 1;
+	MY(alpha_min) = 50;
+	MY(alpha_max) = 145;
+	MY(alpha_fade) = 200;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 4;
+	MY(gravity) = 0.4;
+	MY(sizeincrease) = -5;
+	MY(size_min) = 2;
+	MY(size_max) = 1;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -150;
+	MY(spin_max) = 150;
+	MY(stretchfactor) = 2;
+	MY(tex_min) = 76;
+	MY(tex_max) = 76;
+	MY(type) = "spark";
+	MY(velocityjitter) = '210.0 210.0 350.0';
+	MY(velocitymultiplier) = 0.2;
+}
+// water mist
+SUB(water_splash_small) {
+	MY(airfriction) = 4;
+	MY(alpha_min) = 100;
+	MY(alpha_max) = 150;
+	MY(alpha_fade) = 256;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 4;
+	MY(gravity) = 0.9;
+	MY(sizeincrease) = 25;
+	MY(size_min) = 6;
+	MY(size_max) = 26;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -15;
+	MY(spin_max) = 15;
+	MY(tex_min) = 75;
+	MY(tex_max) = 75;
+	MY(type) = "smoke";
+	MY(velocityjitter) = '150.0 150.0 250.0';
+	MY(velocitymultiplier) = 0.4;
+}
+
+// large splash of water
+DEF(water_splash_large);
+// main water
+SUB(water_splash_large) {
+	MY(airfriction) = 1;
+	MY(alpha_min) = 100;
+	MY(alpha_max) = 250;
+	MY(alpha_fade) = 600;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 15;
+	MY(gravity) = 0.5;
+	MY(originjitter) = '21.0 21.0 1.0';
+	MY(sizeincrease) = -20;
+	MY(size_min) = 15;
+	MY(size_max) = 25;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -150;
+	MY(spin_max) = 150;
+	MY(tex_min) = 76;
+	MY(tex_max) = 76;
+	MY(type) = "alphastatic";
+	MY(velocityjitter) = '150.0 150.0 150.0';
+	MY(velocitymultiplier) = 0.2;
+}
+// small water drops
+SUB(water_splash_large) {
+	MY(airfriction) = 1;
+	MY(alpha_min) = 50;
+	MY(alpha_max) = 145;
+	MY(alpha_fade) = 200;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 4;
+	MY(gravity) = 0.4;
+	MY(sizeincrease) = -5;
+	MY(size_min) = 2;
+	MY(size_max) = 1;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -150;
+	MY(spin_max) = 150;
+	MY(stretchfactor) = 2;
+	MY(tex_min) = 76;
+	MY(tex_max) = 76;
+	MY(type) = "spark";
+	MY(velocityjitter) = '210.0 210.0 350.0';
+	MY(velocitymultiplier) = 0.3;
+}
+// water mist
+SUB(water_splash_large) {
+	MY(airfriction) = 4;
+	MY(alpha_min) = 100;
+	MY(alpha_max) = 150;
+	MY(alpha_fade) = 256;
+	MY(color_min) = "0xd3e5ea";
+	MY(color_max) = "0xa6eaff";
+	MY(count) = 10;
+	MY(gravity) = 0.9;
+	MY(originjitter) = '21.0 21.0 1.0';
+	MY(sizeincrease) = 35;
+	MY(size_min) = 6;
+	MY(size_max) = 26;
+	MY(startangle_min) = 0;
+	MY(startangle_max) = 360;
+	MY(spin_min) = -15;
+	MY(spin_max) = 15;
+	MY(tex_min) = 75;
+	MY(tex_max) = 75;
+	MY(type) = "smoke";
+	MY(velocityjitter) = '180.0 180.0 250.0';
+	MY(velocitymultiplier) = 0.4;
+}
\ No newline at end of file
